dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
I wonder if we could identify the `default` artifact in a bit more straightforward way (now we rely it's at the end of the list, has 'jar' in the name, etc.). I don't know if it's possible.

Or if using the same logic - maybe can be implemented a bit shorter, e.g. something like:
``` groovy
remove(configurations.archives.artifacts.archives.reverse.find { it.getType() == 'jar' })
```